docs/statistics.md

```diff
@@ -1,269 +1,425 @@
+| byteMSS                 | instantaneous     | bytes               | ✓                    | ✓                      | int       |
+| pktSndBuf               | instantaneous     | packets             | ✓                    | -                      | int       |
+| byteSndBuf              | instantaneous     | bytes               | ✓                    | -                      | int       |
+| msSndBuf                | instantaneous     | ms (milliseconds)   | ✓                    | -                      | int       |
+| msSndTsbPdDelay         | instantaneous     | ms (milliseconds)   | ✓                    | -                      | int       |
+| pktRcvBuf               | instantaneous     | packets             | -                    | ✓                      | int       |
+| byteRcvBuf              | instantaneous     | bytes               | -                    | ✓                      | int       |
+| msRcvBuf                | instantaneous     | ms (milliseconds)   | -                    | ✓                      | int       |
+| msRcvTsbPdDelay         | instantaneous     | ms (milliseconds)   | -                    | ✓                      | int       |
+
+
+# Detailed Description
+
+## Accumulated Statistics
+
+### msTimeStamp
+
+The time elapsed since the SRT socket has been created (after successful call to `srt_connect(...)` or `srt_bind(...)` function), in milliseconds. Available both for sender and receiver.
+
+### pktSentTotal
+
+The total number of sent data packets, including retransmitted packets. Available for sender.
+
+### pktRecvTotal
+
+The total number of received packets, including retransmitted packets. Available for receiver.
+
+### pktSndLossTotal
+
+The total number of data packets considered or reported as lost at the sender side. Does not correspond to the packets detected as lost at the receiver side. Available for sender.
+
+A packet is considered lost in two cases: 
+1. Sender receives a loss report from a receiver,
+2. Sender initiates retransmission after not receiving an ACK packet for a certain timeout. Refer to `FASTREXMIT` and `LATEREXMIT` algorithms.
+
+### pktRcvLossTotal
 
-## msTimeStamp
+TODO: 
 
-Time elapsed since the SRT socket was started (after successful call to `srt_connect(...)` or `srt_bind(...)`), in milliseconds.
+- Update the description.
+- Think whether we are going to split lost and reordered packets here and provide user a formula how to get lost only packets if not
 
-## pktSentTotal
+The total number of data packets detected as lost at the receiver side. Available for receiver.
 
-The total number of sent data packets, including retransmitted packets. Applicable for data sender.
+If a packet is received out of order, the gap (sequence discontinuity) is also taken into account and the number of lost packets is increased by the discontinuity size independently of the reordering tolerance value.
```

Comment:
I'd hold up this description until it's clarified by fixing #899. The current state is kinda wrong and reordered packets are now counted as lost. By mentioning the number of reordered packets though you gave me an idea. The missing sequences can be first collected as reordered packets (if reorder tolerance is working), and then if they turn out to not arrive when the reorder tolerance elapses, they would be "re-qualified" as lost (that is, number of these packets would be subtracted from reordered stat and added to loss stat).

Reply:
I created a ticket #1212 for this. I honestly have a lot of concerns regarding this statistics. If you can described in more details how the reorder tolerance is taken into account, it will help. But as I described in the ticket, the reordered packets should be calculated independently on the reorder tolerance. They are simple in-order or out-of-order. Please add the details in the ticket, not here. I'll resolve the current conversation.

Reply:
Generally you have a handler for every incoming packet. If a jump-over is seen, and your current reorder tolerance is > 0, this gap is notified in the "fresh loss" record for now. After so many packets pass as the reorder tolerance and the packet still didn't arrive, only then is it treated as lost and LOSSREPORT is sent.

The reorder tolerance is a dynamic value, however. It may happen that the packet was reordered, but it was treated as lost because it didn't arrive before the tolerance elapsed (or it's 0 now). But if a belated packet with R=0 is seen, then the distance between its sequence and top receiver sequence is set as the new reorder tolerance (but only up to maximum LOSSMAXTTL value).

The result is that it simply gives it time to receive the reordered packet without making a noise of a lost packet, but with some limits, so it still may happen that some packet will arrive with a greater reorder distance, and therefore will be treated mistakenly as lost.

But then, the method I suggested last time should work. The reorder prevention mechanism must have some limits in which it tolerates the reordering - but statistics don't need it. They can extend the tolerance for packet reception even in infinity - let's say up to the ACK - so that a packet that arrived past the tolerance as reordered, but it was already loss-reported, will still be counted as reordered, not as lost.